routes/web.php

```diff
@@ -47,7 +47,8 @@
         Route::get('users/{user}/edit', 'edit')->name('users.edit');
         Route::patch('users/{user}', 'update')->name('users.update');
         Route::get('users/{user}/chart', 'chart')->name('users.chart');
-        Route::get('users/{user}/tree', 'tree')->name('users.tree');
+        Route::get('users/{user}/tree/vertical', 'treeVertical')->name('users.tree-vertical');
```

Comment:
Halo mas, di sini kita tetap perlu default action untuk endpoint `/tree`
```suggestion
        Route::get('users/{user}/tree', 'treeVertical')->name('users.tree');
        Route::get('users/{user}/tree/vertical', 'treeVertical')->name('users.tree-vertical');
```